src/shaders/circle.fragment.glsl

```diff
@@ -2,6 +2,8 @@ varying vec3 v_data;
 
 #pragma mapbox: define highp vec4 color
 #pragma mapbox: define mediump float radius
+#pragma mapbox: define mediump float ang_start
```

Comment:
I would rename `ang_start` and `ang_end` to their full name `angle_start` and `angle_end` to be more explicit.